amethyst_core/src/hidden.rs

```diff
@@ -14,9 +14,32 @@ impl Component for Hidden {
 
 /// Like [Hidden](struct.Hidden.html), but can propagate through children when the [HideHierarchySystem](struct.HideHierarchySystem.html)
 /// is enabled in the [RenderBundle](struct.RenderBundle.html).
-#[derive(Clone, Debug, Default)]
-pub struct HiddenPropagate;
```

Comment:
This is a breaking change, right? I think this needs a mention in the ChangeLo
g, and possibly examples